Description: 
 Fix FTBFS due to lack of #include <sys/stat.h>. Without these #includes,
 we get a link time failure due to undefined S_ISCHR.
Author: Alex Chiang <achiang@canonical.com>
Bug: https://sourceforge.net/tracker/?func=detail&aid=3105671&group_id=2599&atid=102599

Index: zvbi-0.2.33/contrib/ntsc-cc.c
===================================================================
--- zvbi-0.2.33.orig/contrib/ntsc-cc.c	2010-11-09 09:22:16.000000000 +0100
+++ zvbi-0.2.33/contrib/ntsc-cc.c	2010-11-09 09:22:55.000000000 +0100
@@ -34,6 +34,7 @@
 #include <locale.h>
 #include <sys/ioctl.h>
 #include <sys/types.h>
+#include <sys/stat.h>
 #include <sys/time.h>
 #ifdef HAVE_GETOPT_LONG
 #  include <getopt.h>
Index: zvbi-0.2.33/src/io-dvb.c
===================================================================
--- zvbi-0.2.33.orig/src/io-dvb.c	2010-11-09 09:22:16.000000000 +0100
+++ zvbi-0.2.33/src/io-dvb.c	2010-11-09 09:22:55.000000000 +0100
@@ -29,6 +29,8 @@
 #include <errno.h>
 #include <sys/select.h>
 #include <sys/ioctl.h>
+#include <sys/types.h>
+#include <sys/stat.h>
 
 #ifndef HAVE_S64_U64
   /* Linux 2.6.x asm/types.h defines __s64 and __u64 only
Index: zvbi-0.2.33/src/io-v4l.c
===================================================================
--- zvbi-0.2.33.orig/src/io-v4l.c	2010-11-09 09:22:16.000000000 +0100
+++ zvbi-0.2.33/src/io-v4l.c	2010-11-09 09:22:55.000000000 +0100
@@ -42,6 +42,7 @@
 #include <assert.h>
 #include <sys/time.h>		/* timeval */
 #include <sys/types.h>		/* fd_set, uid_t */
+#include <sys/stat.h>		/* S_ISCHR */
 #include <sys/ioctl.h>		/* for (_)videodev.h */
 #include <pthread.h>
 
